src/components/step-sequence/index.js

```diff
@@ -0,0 +1,4 @@
+import StepSequence from './step-sequence.component';
+import StepSequenceItem from './step-sequence-item';
+
+export { StepSequence, StepSequenceItem };
```

Comment:
I think that we don't need to import and then export components. I think we can just export 
```
export { default as StepSequence } from './step-sequence.component'
export { default as StepSequenceItem } from './step-sequence-item' 
```
What do you think about this idea